rem.go

```diff
@@ -405,17 +424,23 @@ func promptBool(promptStr string) (yes bool) {
 }
 
 func argsHaveOption(long string, short string) (hasOption bool, foundAt int) {
-	for i, arg := range os.Args {
-		if arg == "--"+long || arg == "-"+short {
-			return true, i
-		}
+	hasOption, foundAt = argsHaveOptionExplicit("-" + short)
```

Comment:
bug: this means `argsHaveOption` will always prefer the short option instead of the one that comes last